lumigator/python/mzai/backend/backend/settings.py

```diff
@@ -81,9 +81,15 @@ def inherit_ray_env(self, runtime_env_vars: Mapping[str, str]):
             if env_var is not None:
                 runtime_env_vars[env_var_name] = env_var
 
-    OAI_API_KEY: str = os.environ.get("OPENAI_API_KEY", "")
+    @computed_field
+    @property
+    def OAI_API_KEY(self) -> str:  # noqa: N802
+        return os.environ.get("OPENAI_API_KEY", "")
```

Comment:
Maybe we can add `info` or `warning` logs here to notify the user about the presence or absence of the environment variable. This would make it easier to track these kinds of issues in the future. Same for the Mistral key. WDYT?

Reply:
added here at the service level https://github.com/mozilla-ai/lumigator/pull/599/files#diff-35d12bc5fa5da1376ba28fa5fe007bfd8849aa4229dd39f85125969c2134d253R50